mix.exs

```diff
@@ -81,10 +81,6 @@ defmodule Bokken.MixProject do
       {:telemetry_poller, "~> 1.0"},
       {:phoenix_live_dashboard, "~> 0.5"},
 
-      # admin panel
-      {:phoenix_html, "~> 2.11", override: true},
```

Comment:
Isn't `phoenix_html` needed by `phoenix_swoosh`, in order to render HTML?
Source: https://github.com/swoosh/phoenix_swoosh
![image](https://github.com/coderdojobraga/bokken/assets/76881129/6efe1ede-72da-4058-b44c-da311ba3a280)

Reply:
Yes, you are right :muscle: 

Fixed on last push :+1: